### PR TITLE
add tier 1-3 scorm icons

### PR DESCRIPTION
**Pull request by tinglisD2L**
These icons are designed by Sarah Bracewell. There was a different Tier 3 SCORM icon added a while ago (https://github.com/BrightspaceUI/icons/pull/192), but Sarah said the new icons should be used.

This is my first time working in this repo. I just added the SVG icons and ran `npm run build`. Not sure if it's the correct workflow. Thanks in the advance for your help!

### CODE REVIEW

**Comment by dlockhart** [comment]
Thanks @tinglisD2L. Have a look at the [contribution docs](https://github.com/BrightspaceUI/icons#updating-or-contributing-new-icons) as there's a list of steps to go through to make sure the SVG is formatted correctly. Looks like a couple of these are missing.

File: tier2-icons.html [hunk 1]
```diff
@@ -529,13 +529,17 @@
...
         </g>
         <g id="seating">
           <path d="M20 12h-3V9.54c.898-.153 1.783-.37 2.65-.65a2 2 0 0 0 1.07-2.92l-3-5a2.058 2.058 0 0 0-1.27-.92A2.3 2.3 0 0 0 16 0a1.926 1.926 0 0 0-1.06.31 6.142 6.142 0 0 1-2.87.69A7.148 7.148 0 0 1 8.89.21a2 2 0 0 0-2.61.76l-3 5A2.008 2.008 0 0 0 4.4 8.91c.854.264 1.722.48 2.6.65V12H4a2.006 2.006 0 0 0-2 2v2a2.006 2.006 0 0 0 2 2h1v4.7A1.3 1.3 0 0 0 6.3 24h.4A1.3 1.3 0 0 0 8 22.7V22h8v.69A1.315 1.315 0 0 0 17.31 24h.38A1.315 1.315 0 0 0 19 22.69V18h1a2.006 2.006 0 0 0 2-2v-2a2.006 2.006 0 0 0-2-2zM5 7l3-5c1.26.648 2.654.99 4.07 1A7.8 7.8 0 0 0 16 2l3 5a24.054 24.054 0 0 1-7.11 1.02A23.2 23.2 0 0 1 5 7zm10 2.84V12H9V9.86a26.752 26.752 0 0 0 6-.02zM16 20H8v-2h8v2zm4-4H4v-2h16v2z" fill-rule="evenodd"/>
         </g>
-        <g id="self-assessment">
+        <g id="self-assessment" mirror-in-rtl="true">
```
Comment (dlockhart) [inline tier2-icons.html]: These additional "mirror-in-rtl" entries were my fault -- I modified the SVGs a little while ago without re-running the build, so your build is picking them up. They're OK to go in, so no change on your part required. Sorry about that!

File: d2l-icons.md [hunk 2]
```diff
@@ -38,102 +38,103 @@ Size: `18px` x `18`px
...
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/draft.svg?raw=true) | draft | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/pin-filled.svg?raw=true) | pin-filled | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/volume.svg?raw=true) | volume |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/edit.svg?raw=true) | edit | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/pin-hollow.svg?raw=true) | pin-hollow | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/wizard.svg?raw=true) | wizard |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/email-open.svg?raw=true) | email-open | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/preview.svg?raw=true) | preview | | &nbsp; | &nbsp; |
+| ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/email.svg?raw=true) | email | | ![](https://rawgit.com/BrightspaceUI/icons/master/images/tier2/print.svg?raw=true) | print | | &nbsp; | &nbsp; |
 
 ## tier3
```
Comment (tinglisD2L) [inline d2l-icons.md]: @dlockhart Hi Dave. I just noticed that `d2l-icons.md` didn't get updated for Tier3 even though I did replace an icon. So now the preview page still shows the old icon. What should I do?

Comment (dlockhart) [reply d2l-icons.md]: Interesting, it should get updated as part of the `npm run build`. Could you try running it again? Not the end of the world if it doesn't get updated, but curious nonetheless.

Comment (tinglisD2L) [reply d2l-icons.md]: The link https://rawgit.com/BrightspaceUI/icons/master/images/tier3/scorm.svg?raw=true in this file is already pointing to the current icon. But when I inspect the icon used in https://github.com/BrightspaceUI/icons/blob/master/d2l-icons.md, it's pointing to https://camo.githubusercontent.com/2c3ee8597fa2ac1b3f38ecdd71502fe7bcd13fbe/68747470733a2f2f7261776769742e636f6d2f427269676874737061636555492f69636f6e732f6d61737465722f696d616765732f74696572332f73636f726d2e7376673f7261773d74727565, which shows the old icon.

Comment (tinglisD2L) [reply d2l-icons.md]: Hold up! It just updated. I was just too impatient. It's all good now :)

**Pull request merged**